rfc/text/advanced/ap_10_rpc_progressive_calls.md

```diff
@@ -1,5 +1,249 @@
+calls to the same *Callee*.
+
+If binding all ongoing `progressive` calls to the same *Callee* is not required, *Caller* can set `sticky` option to `FALSE`. 
+
+If `CALL.Options.sticky` is not specified it is treated like `TRUE`, so all `progressive`
+calls go to the same *Callee*.
+
+**Progressive Call Cancellation**
+
+If the original *Caller* is no longer available for some reason (has left the realm), then the *Dealer* will send 
+an `INTERRUPT` to the *Callee*. The `INTERRUPT` will have `Options.mode` set to `"killnowait"` to indicate to the 
+client that no response should be sent to the `INTERRUPT`.
+
+```
+[INTERRUPT, INVOCATION.Request|id, Options|dict]
+```
+Options:
+```
+INTERRUPT.Options.mode|string == "killnowait"
+```
+
+Progressive call cancellation, like progressive call result cancellation closes an important safety gap: 
+In cases where progressive calls are used to stream data from *Callers*, and network connectivity is unreliable, 
+*Callers* my often get disconnected in the middle of sending progressive data. This can lead to unneeded memory
+consumption on *Dealer* and *Callee* sides, because of storing runtime data about ongoing calls.
+
+The message flow for progressive results cancellation involves:
+
+{align="left"}
+,------.            ,------.                 ,------.
+|Caller|            |Dealer|                 |Callee|
+`--+---'            `--+---'                 `--+---'
+   |  CALL (progress)  |                        |
+   | ----------------->|                        |
+   |                   | INVOCATION (progress)  |
+   |                   | ---------------------->|
+   |  CALL (progress)  |                        |
+   | ----------------->|                        |
+   |                   | INVOCATION (progress)  |
+   |                   | ---------------------->|
+,--+---.               |                        |
+|Caller|               |                        |
+`------'               |                        |
+ (gone)                |                        |
+                       |       INTERRUPT        |
+                       | ---------------------->|
+                       |                        |
+                    ,--+---.                 ,--+---.
+                    |Dealer|                 |Callee|
+                    `------'                 `------'
+
+
+Note: Any `ERROR` returned by the *Callee*, in response to the `INTERRUPT`, is ignored (same as in regular call 
+canceling when mode="killnowait"). So, it is not necessary for the *Callee* to send an `ERROR` message.
+
+**Ignoring Progressive Call Requests**
+
+Unlike other advanced features, this one can not be omitted by *Callee*.
+So if *Callee* doesn't support this feature, *Dealer* MUST respond to *Caller* with 
+`wamp.error.feature_not_supported` error message. 
+
+A *Callee* that supports progressive calls, but does not support call canceling is considered by the *Dealer* to 
```

Comment:
but does not support call cancelling, shall be considered by the *Dealer* as not supporting progressive calls